### PR TITLE
Feature/18239 school data import

### PR DESCRIPTION
**Pull request by GuyHarwood**
Add 60 pupils to each of the 5 schools in UAT only

### CODE REVIEW

File: admin/data/sql/migrations/062.do.school-dfeNumber-non-unique.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DROP INDEX mtc_admin.school.school_dfeNumber_uindex
+ALTER TABLE mtc_admin.school ALTER COLUMN dfeNumber NVARCHAR(20) NOT NULL
+CREATE UNIQUE INDEX school_dfeNumber_uindex ON mtc_admin.school (dfeNumber)
```
Comment (jon-shipley) [inline admin/data/sql/migrations/062.do.school-dfeNumber-non-unique.sql]: Why would dfeNumber be an varchar rather than an int?

Comment (GuyHarwood) [reply admin/data/sql/migrations/062.do.school-dfeNumber-non-unique.sql]: because the data dept now prefix academy trusts with `MAT`

Comment (jon-shipley) [reply admin/data/sql/migrations/062.do.school-dfeNumber-non-unique.sql]: That's odd. So they are encoding two pieces of data in a single column?  I hope the remaining dfeNumber is unique.  If so, we could have new column called `isMAT` and parse it at load time.  I would prefer that query to find MATs than a prefix search on an nvarchar.

Comment (GuyHarwood) [reply admin/data/sql/migrations/062.do.school-dfeNumber-non-unique.sql]: they also duplicate dfeNumbers while a school transitions to an academy.  This is a whole topic we should discuss privately.  Given the upcoming trial only contains a subset of schools, i think we can potentially sidestep most of the pain for now.  But it does need actioning during March-June

Comment (GuyHarwood) [reply admin/data/sql/migrations/062.do.school-dfeNumber-non-unique.sql]: @jon-shipley ive removed this migration for now, as the school data will be done separately

File: admin/data/sql/not-used-reference/062.do.school-dfeNumber-non-unique.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+DROP INDEX mtc_admin.school.school_dfeNumber_uindex
+ALTER TABLE mtc_admin.school ALTER COLUMN dfeNumber NVARCHAR(20) NOT NULL
+CREATE UNIQUE INDEX school_dfeNumber_uindex ON mtc_admin.school (dfeNumber)
```
Comment (GeorgeChatzigiannis) [inline admin/data/sql/not-used-reference/062.do.school-dfeNumber-non-unique.sql]: new line needed

File: admin/data/sql/not-used-reference/062.undo.school-dfeNumber-non-unique.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+DROP INDEX mtc_admin.school.school_dfeNumber_uindex
+ALTER TABLE mtc_admin.school ALTER COLUMN dfeNumber INT NOT NULL
+CREATE UNIQUE INDEX school_dfeNumber_uindex ON mtc_admin.school (dfeNumber)
```
Comment (GeorgeChatzigiannis) [inline admin/data/sql/not-used-reference/062.undo.school-dfeNumber-non-unique.sql]: new line needed

File: admin/data/sql/not-used-reference/062.do.school-dfeNumber-non-unique.sql (hunk 2 above)
Comment (GuyHarwood) [reply admin/data/sql/not-used-reference/062.do.school-dfeNumber-non-unique.sql]: done

File: admin/data/sql/not-used-reference/062.undo.school-dfeNumber-non-unique.sql (hunk 3 above)
Comment (GuyHarwood) [reply admin/data/sql/not-used-reference/062.undo.school-dfeNumber-non-unique.sql]: done

**Comment by GuyHarwood** [comment]
@GeorgeChatzigiannis Migration 62 is the latest.  Those files are in a sub directory for reference.
I will remove the migration number to avoid confusion.

**Pull request merged**